docs/user-guide/gaffer-basics/what-is-gaffer.md

```diff
@@ -21,4 +22,24 @@ flowchart TD
         B --> C(Gaffer)
         C --> D(key-value store)
     end
-```
\ No newline at end of file
+```
+
+!!! note
+    Knowledge of the [Gaffer schema](../schema.md) in use is usually required to
+    formulate a graph query but technically optional hence the dotted connection.
+
+## Why would you want to use Gaffer?
+
+Conceptually a graph database/data platform can have a large variety of uses
+where it'll provide specific advantages over other data storage/analytic
+platforms which will be discussed in the ["What is Graph?"](./what-is-a-graph.md)
```

Comment:
```suggestion
platforms which is discussed in the ["What is Graph?"](./what-is-a-graph.md)
```
Even if this isn't completed yet, we don't want to have to come back and change this page I don't think.